src/utilities/generic.ts

```diff
@@ -390,6 +390,10 @@ async function* streamAsyncIterable(stream: ReadableStream) {
   }
 }
 
+export function unique(list: any[]) {
```

Comment:
Would 
`export` function unique<T>(list: T[]) { ... }`
be sufficient?

Reply:
I think that works! Just out of curiosity, is there a concern for defaulting to string?

Reply:
Not necessarily, but it seems misleading to offer such default because it hasn't really been used for strings; the types it is offered for vary!

Reply:
Oh. I was defaulting it to string because I saw that the usage was with string arrays. Also, this would only work with arrays of primitives. The current implementation and my suggestion will not properly dedupe arrays of objects. Objects would need to be stringified in order to properly do the comparison. I noticed that in the `Row.svelte` component, it's being used to dedupe `matchingEvents`